### PR TITLE
start implementing handlers

### PR DESCRIPTION
**Pull request by danylozhornyk**

### CODE REVIEW

File: server/handlers/thread_handler.c [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#include "../inc/server.h"
+
+// This function retrieves the client request from the SSL connection
+char* get_client_request(SSL* ssl, int length) {
```
Comment (Maksull) [inline server/handlers/thread_handler.c]: Make it static, it used only in thread_handler function

File: server/handlers/thread_handler.c [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    }
+
+    buffer[bytesRead] = '\0';
+
+    // Return a duplicate of the buffer string
+    return mx_strdup(buffer);
+}
+
+//  Function reads data from the client through the provided SSL connection
+char* read_client_data(SSL *ssl) {
```
Comment (Maksull) [inline server/handlers/thread_handler.c]: Make it static, it used only in thread_handler function

File: server/server_utils/requests.c [hunk 3]
```diff
@@ -0,0 +1,43 @@
+#include "../inc/server.h"
+
+// Get the type of the request from the read json object
+t_request_type get_request_type(cJSON* json) {
```
Comment (Maksull) [inline server/server_utils/requests.c]: Make it static and remove from header

File: server/server_utils/responses.c [hunk 4]
```diff
@@ -0,0 +1,37 @@
+#include "../inc/server.h"
+
+// Get a string-formatted error response from a created json object
+char* get_json_response_for(t_response_code error_code, t_request_type req_type) {
```
Comment (Maksull) [inline server/server_utils/responses.c]: Make it static and remove from header

**Review by Maksull — CHANGES_REQUESTED** [review]
Make some functions static and remove them from header. Also check files' last lines. They all should have an empty line

File: server/server_utils/requests.c (hunk 3 above)
Comment (danylozhornyk) [reply server/server_utils/requests.c]: it will be used in some other funcs in future so it should stay as it is

**Review by Maksull — APPROVED** [review]

**Pull request merged**